demo/lib/orchestrator.go

```diff
@@ -586,13 +583,15 @@ func (e *Orchestrator) PrintLogs() {
 func (e *Orchestrator) Shutdown() {
 	fmt.Println("[+] Shutdown all nodes")
 	for _, no := range e.nodes {
+		no := no
```

Comment:
Is this really required? The function below is already copying the `node.Node`, no?

Reply:
Same problem, we don't know the type behind the `Node` interface.